### PR TITLE
Fable2

### PR DESCRIPTION
**Pull request by OlegZee**
Please look into this PR which makes the library compatible with Fable2 compiler.
Notice I split it onto three commits for easier review.

The latter issue. I don't like the approach however I got no more ideas for doing this right. Asked people in Fable gitter but got nothing.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,8 @@
 // Place your settings in this file to overwrite default and user settings.
 {
+    "workbench.colorCustomizations": {
```
Comment (ncthbrt) [inline .vscode/settings.json]: Can we omit this from the review please.

**Comment by ncthbrt** [comment]
@OlegZee Do we need to also get rid of the ionide symbol cache and it to .gitignore.

**Comment by OlegZee** [comment]
Btw notice it's using Thoth.Json so I should also update Suave project. I just forgot this.
Please do not accept this PR before I push more changes (tomorrow).

**Comment by OlegZee** [comment]
@ncthbrt  I'm done upgrading to Thoth in server project. Once we complete with this project (I would need a published nuget package to reference it) I will make sure the sample in Elmish project are functioning.

**Comment by OlegZee** [comment]
@ncthbrt could you please publish it to nuget so that I can complete migration (make PR) for Fable.Websockets.Elmish?

**Comment by ncthbrt** [comment]
Apologies. I'm away for the weekend, do will likely only be able to get
around to this on Monday at the earliest.

**Comment by ncthbrt** [comment]
@OlegZee I've published both packages to nuget.

**Comment by OlegZee** [comment]
Great! Will prepare another PR tomorrow.

вт, 18 июня 2019 г. в 1:00, Nick Cuthbert <notifications@github.com>:

**Comment by OlegZee** [comment]
@ncthbrt any update on the other PR?

**Pull request merged**